channels/rail/server/rail_main.c

```diff
@@ -297,8 +297,6 @@ static UINT rail_send_server_handshake_ex(RailServerContext* context,
 	if (!context || !handshakeEx || !context->priv)
 		return ERROR_INVALID_PARAMETER;
 
-	/* Save channel flags to context */
-	context->priv->channelFlags = handshakeEx->railHandshakeFlags;
```

Comment:
Maybe I´m missing something, but here we should update the flags as these are the ones the client is allowed to use?